internal/commands/ip_access/update_inputs.go

```diff
@@ -0,0 +1,37 @@
+package ipaccess
+
+import (
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
+type updateInputs struct {
+	cli.ProjectInputs
+	IPAddress    string
+	NewIPAddress string
+	Comment      string
+}
+
+func (i *updateInputs) Resolve(profile *user.Profile, ui terminal.UI) error {
+	if err := i.ProjectInputs.Resolve(ui, profile.WorkingDirectory, false); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (i *updateInputs) resolveAllowedIP(ui terminal.UI, allowedIPs []realm.AllowedIP) (realm.AllowedIP, error) {
+	if len(i.IPAddress) > 0 {
+		for _, allowedIP := range allowedIPs {
+			if allowedIP.IPAddress == i.IPAddress {
+				return allowedIP, nil
+			}
+		}
+		return realm.AllowedIP{}, fmt.Errorf("unable to find allowed IP: %s", i.IPAddress)
+	}
+
+	return realm.AllowedIP{}, nil
```

Comment:
if we make it here, we should either error or prompt the user for more info (rather than rely on a failed api call for an error)